iidm/iidm-api/src/main/java/com/powsybl/iidm/network/TapChanger.java

```diff
@@ -22,6 +22,17 @@ public interface TapChanger<
     R extends TapChangerStepsReplacer<R, A>,
     A extends TapChangerStepAdder<A, R>> {
 
+    /**
+     * Get the load tap changing capabilities status.
+     */
+    boolean hasLoadTapChangingCapabilities();
+
+    /**
+     * Set the load tap changing capabilities status.
+     * @return itself for method chaining.
+     */
+    C setLoadTapChangingCapabilities(boolean status);
```

Comment:
Can you rename the parameter here?
`status` -> `loadTapChangingCapabilities`

(The name was already changed in the implementations.)